src/Taxjar/TaxjarApi.cs

```diff
@@ -472,5 +475,26 @@ public virtual async Task<List<SummaryRate>> SummaryRatesAsync()
             var response = await SendRequestAsync<SummaryRatesResponse>("summary_rates", null, Method.GET).ConfigureAwait(false);
             return response.SummaryRates;
         }
+
+        private string getUserAgent()
+        {
+            #if NET452
+                string platform = Environment.OSVersion.VersionString;
```

Comment:
I would leave a comment here indicating the reason for this conditional (the references from our messages would be fine).